.travis.yml

```diff
@@ -39,6 +42,7 @@ before_script:
   - ${GNUPG_BINARY:-gpg} --version
   - mkdir gpg_keyring
 script:
+  - export PYTHONWARNINGS=all
```

Comment:
I could be wrong, but I don't think `all` is a valid value for this environment variable.
See here: https://docs.python.org/3/using/cmdline.html#envvar-PYTHONWARNINGS
Can you point me to where `all` is supported?

Reply:
It seams as this has changed in Python 3.7, which I was not aware of until now. In Python 3.6 there [is still the value ```all```.](https://docs.python.org/3.6/using/cmdline.html#cmdoption-w.) There is a mention about it in the [Python 3.7 release notes.](https://docs.python.org/3/whatsnew/3.7.html#warnings) As ```always``` is also supported, but not documented, in Python 3.6 I will change it to ```always```.

Reply:
That situation is...not great. Using `always` sounds like the way to go. Thanks for changing it!